packages/react/src/components/DateTimePicker/DateTimePickerV2.test.jsx

```diff
@@ -540,6 +540,38 @@ describe('DateTimePickerV2', () => {
     expect(screen.queryByRole('dialog')).toBeNull();
   });
 
+  it('should disable button when clear time picker input in single select', () => {
+    const { i18n } = DateTimePicker.defaultProps;
+    render(
+      <DateTimePicker
+        {...dateTimePickerProps}
+        useNewTimeSpinner
+        onApply={jest.fn()}
+        datePickerType="single"
+        dateTimeMask="YYYY-MM-DD hh:mm A"
+        hasTimeInput
+        showRelativeOption={false}
+        defaultValue={{
+          timeRangeKind: PICKER_KINDS.SINGLE,
+          timeSingleValue: {
+            startDate: '2020-04-01',
+            startTime: '12:34 AM',
+          },
+        }}
+      />
+    );
+
+    // default value is 2020-04-01 12:34 AM
+    expect(screen.getByText('2020-04-01 12:34 AM')).toBeVisible();
+
+    // first open the menu
+    userEvent.click(screen.getAllByText(/2020-04-01 12:34 AM/i)[0]);
+
+    userEvent.clear(screen.getByTestId('date-time-picker-input'));
+
+    expect(screen.getByText(i18n.applyBtnLabel)).toBeDisabled();
```

Comment:
Are we saying that if we enter a date/time then we can never clear it?
I suspect that this component needs to have a "required" attribute.  If the date is required, it should not allow the apply if the value is empty.  BUt if the component is not required, then I should be able to clear the date.  

@JordanWSmith15 @davidicus

Reply:
> Are we saying that if we enter a date/time then we can never clear it? I suspect that this component needs to have a "required" attribute. If the date is required, it should not allow the apply if the value is empty. BUt if the component is not required, then I should be able to clear the date.
> 
> @JordanWSmith15 @davidicus

You can either clear the input manually or click on the `Clear` button as Jordan mentioned above. The test was saying if time input is cleared/empty we need to disable the `Apply` button